### PR TITLE
docs: add etcd deployment

### PR DESCRIPTION
**Pull request by nicecui**
This PR adds the description of etcd cluster deployment. More operation content about kubernetes will be written in the future PRs.

### CODE REVIEW

File: docs/user-guide/operations/kubernetes.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
 # Kubernetes
 
-## GreptimeDB Operator
+Besides Kubernetes comand line tool `kubectl`, `helm` and `gtctl` can also be used to manage GreptimeDB clusters.
```
Comment (daviderli614) [inline docs/user-guide/operations/kubernetes.md:3]: `kubectl`这个字段觉得可以去掉，昨天看错了，一共两种方式按照，一种`helm`，一种`gtctl`

File: docs/user-guide/operations/kubernetes.md [hunk 2]
```diff
@@ -56,20 +61,35 @@ helm install gtcloud greptime/greptimedb-operator -n default --devel
 
 The maintained Helm charts are in [helm-charts][6].
 
-### 3\. Create your own GreptimeDB cluster
+### 3. Create your own GreptimeDB cluster
```
Comment (daviderli614) [inline docs/user-guide/operations/kubernetes.md:64]: 可以把etcd和greptimedb分开两点进行安装：
### 3. Create your own etcd cluster

Create etcd cluster:

```shell
helm install etcd greptime/greptimedb-etcd -n default --devel
```
---

### 4. Create your own GreptimeDB cluster

Create GreptimeDB cluster:

```shell
helm install mydb greptime/greptimedb -n default --devel
```

If you already have the etcd cluster, you can configure the etcd cluster:

```shell
helm install mycluster greptime/greptimedb -set etcdEndpoints=<your-etcd-cluster-endpoints> \
-n default --devel
```
---

**Review by daviderli614 — APPROVED** [review]
LGTM

File: docs/user-guide/operations/kubernetes.md [hunk 3]
```diff
@@ -1,12 +1,10 @@
 # Kubernetes
+<!-- TODO how to apply yaml config -->
+<!-- Besides Kubernetes comand line tool `kubectl`, `helm` and `gtctl` can also be used to manage GreptimeDB clusters.
 
-## GreptimeDB Operator
-
-The GreptimeDB [Operator][1] manages GreptimeDB clusters on [Kubernetes][2] by using [Operator
-pattern][3].
+## kubectl
```
Comment (xtang) [inline docs/user-guide/operations/kubernetes.md:5]:
```suggestion
## Kubectl
```

**Comment by xtang** [comment]
@nicecui almost LGTM except for some typo

**Comment by nicecui** [comment]
@xtang fixed and that all in the comment lines 😂，sharp eyes 👍

**Pull request merged**